### PR TITLE
Setting up travis config for dbmigrations

### PR DESCRIPTION
**Pull request by ibarrae**
Added migrations.yaml to build dbmigrations and updated commands in CI's configs.

Fixed travis and circle config files, also added cache to circle config

Fixed dependency in gender of initial migration

Added environment variables for haskell image

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    PGUSER: ubuntu
+    PGPASS: ""
+    PGDATABASE: snap
+
+version: 2
+jobs:
+  project:
+    docker:
+      - <<: *haskell
+      - <<: *postgres
```
Comment (dbalseiro) [inline .circleci/config.yml:26]: Do you know what this means?

File: migrations/gender.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Description: Update gender constraints for snap-user table.
```
Comment (dbalseiro) [inline migrations/gender.txt:1]: Please, change the name of the file to include a timestamp on its name... so when you have more than one migration, you can list it in order... for example:
2018-07-25-15-45-23_create_type_gender.txt

File: .travis.yml [hunk 3]
```diff
@@ -10,13 +10,22 @@ ghc: "8.2.2"
 cache:
   directories:
     - $HOME/.cabal/
+    - $HOME/.stack/
```
Comment (dbalseiro) [inline .travis.yml:13]: You don't need travis anymore...

**Review by dbalseiro — APPROVED** [review]
I left a couple of comments... please, take care of them and then ship it... Good job!

File: .circleci/config.yml (hunk 1 above)
Comment (ibarrae) [reply .circleci/config.yml:26]: If I am not wrong, does it mean to use the containers named "haskell" and "postgres" defined in the start of `config.yml`

File: migrations/gender.txt (hunk 2 above)
Comment (ibarrae) [reply migrations/gender.txt:1]: @dbalseiro I was wondering if that is not the same as specifing dependency on each migration, like in `gender.txt` is `Depends: initial`

Comment (dbalseiro) [reply migrations/gender.txt:1]: No... it's not

File: .circleci/config.yml (hunk 1 above)
Comment (dbalseiro) [reply .circleci/config.yml:26]: Actually they are like variables of sorts... why are you referencing a variable if you are not using it in more than one place?

File: migrations/gender.txt (hunk 2 above)
Comment (dbalseiro) [reply migrations/gender.txt:1]: here is how migrations looks like in a project:
screenshot from 2018-07-26 09-39-47

**Pull request merged**